CleverTapSDK/InApps/CustomTemplates/SystemAppFunction/CTPushPermissionSystemAppFunction.h

```diff
@@ -0,0 +1,20 @@
+//
+//  CTPushPermissionSystemAppFunction.h
+//  CleverTapSDK
+//
+//  Created by Nishant Kumar on 20/03/25.
+//  Copyright © 2025 CleverTap. All rights reserved.
+//
+
+#ifndef CTPushPermissionSystemAppFunction_h
+#define CTPushPermissionSystemAppFunction_h
+#import "CTCustomTemplate.h"
+#import "CTSystemTemplateActionHandler.h"
+
+@interface CTPushPermissionSystemAppFunction : NSObject
+
++ (CTCustomTemplate *)buildTemplateWithHanlder:(CTSystemTemplateActionHandler *)systemTemplateActionHandler;
```

Comment:
Typo in method name

Reply:
I have updated this typo in app rating PR: https://github.com/CleverTap/clevertap-ios-sdk/pull/415/commits/15195e70fd560143a79e1fa3d2365a0931778599